index.js

```diff
@@ -1,88 +1,125 @@
 // ***** Question 1 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 1", "color: red") 
+console.log("%cQuestion 1", "color: red") 
 
-// console.log(counter) 
-// // => 2
-// console.log("%c----------", "color: red") 
+var counter = 1;
+var counter = 2;
+
+console.log(counter) 
+// => 2
+console.log("%c----------", "color: red") 
```

Comment:
This is really good logic, but i just wanna warn you about the "var" keyword! "Var" is function scoped, which means that you could access this variable from other functions whereas if you use the "let" keyword, it secures your variable as whatever scope you declare it in! Nice job :)